src/ui/pages/Overview/Overview.tsx

```diff
@@ -244,6 +248,29 @@ function CurrentAccountControls() {
   );
 }
 
+function RewardsLinkIcon() {
+  return (
+    <Button
+      kind="ghost"
+      as={UnstyledAnchor}
+      href="https://app.zerion.io/rewards"
```

Comment:
Let's add connection params here!

Reply:
https://github.com/zeriontech/zerion-wallet-extension/pull/657/files#r1836574324

Reply:
Thanks for the reference 👍 Updated ✅
@zerts Could you please verify that I've done the right thing here? :)